docs/reference/language/scripting.md

```diff
@@ -245,6 +259,10 @@ The value in question can be either:
   [element function]($function/#element-functions) that were given when the
   element was constructed.
 
+**Important:** Many fields of element functions are only accessible in a
```

Comment:
```suggestion
Many fields of element functions are only accessible in a
```

Reply:
It stuck out to me because in my opinion it's not clearly more important than the other information in this section. 

Regardless of that, I think this section could be expanded to clarify exactly how it exactly works. Essentially
- fields for required arguments are always accessible
- fields for optional arguments that were provided in the element's constructor are accessible on a constructed element
- fields for optional arguments that weren't mentioned in the constructor are only available when the element is "materialized", this is the case when it's the shown element in a show rule or when the element is returned from a query
- If an element is not materialized, it's fields are not available, also not within a `context` expression